website/docs/style_guide/about_style_guide.md

```diff
@@ -0,0 +1,65 @@
+# Robot Framework Style Guide
+
+## Purpose
+
+This style guide is a community driven set of sensible rules to write your Robot Framework code.
+
+As a starting point the existing standards from [Robot Framework](https://robotframework.org/robotframework/latest/RobotFrameworkUserGuide.html#getting-started) user guide, [Robocop](https://github.com/MarketSquare/robotframework-robocop) and [Robotidy](https://robotidy.readthedocs.io/en/stable/).
+
+## Robocon 2022 Presentation
+
+[Robot Framework is not ...](https://youtu.be/Mpt_4MItha0)
+
+## Contributors
+
+Guido Demmenie, Mannana Koberidze, Kelby Stine
+
+### Special Thanks To
+
+Václav Fuksa, Many Kasiriha, Bartlomiej Hirz, Mateusz Nojek, René Rohner, Miikka Solmela
+
+### We are always looking for contributors
+
+- We meet up once every two weeks to talk about style guide topics
+- Reach out to any one of us on the Slack channels
+
+---
+
+## Basic Rules and Guidelines
+
+Here are some good basic guidelines for formatting your Robot Framework codes. As these are guidelines, use your best judgment in cases not covered by these recommendations.
+
+- Separate pieces of the data, such as keywords and their arguments, with four (4) spaces.
+
+- Robot and Resource file section order:
+
+```robot
+*** Comments ***
+
+*** Settings ***
+
+*** Variables ***
+
+*** Test Cases / Tasks ***
+(not available in resource files)
+
+*** Keywords ***
+```
+
+- Keywords within resource files should be alphabetized.
```

Comment:
I think I would not enforce that. I like to place my keywords based on their order of appearance in the code or by the frequency of usage. I used to alphabetize them, but I don't think we should enforce it. Better to leave it up to the code authors.

Reply:
Because RF sees everything before the first section in a testsuite as comments we added it here. Maybe we should leave it out completely as using the `[Documentation]` setting makes much more sense to add documentation for a testsuite.

Reply:
I think I have a better suggestion.